modules/ROOT/pages/platform/security.adoc

```diff
@@ -6,13 +6,29 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-AuraDB Enterprise and AuraDS Enterprise run within a Virtual Private Cloud (VPC) isolation for your deployment.
+AuraDB Enterprise and AuraDS Enterprise run in a dedicated cloud Account (AWS), Subscription (Azure) or Project (GCP) to achieve complete isolation for your deployment.
 
-The VPC enables you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
 
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or if it should use the cloud provider's internal network. 
+To use the cloud provider's network, create a private endpoint inside your VPC. 
+This gives you a private connection to Aura.
+It's your choice if you want to prevent your Aura instances from being publicly accessible or if you want both public and private access.
```

Comment:
Can't push my suggestion so here it is:
```suggestion
An Aura instance can be publicly available, completely private, or both.

To configure this, you need to be authorized to access to the part of the infrastructure that runs and handles these as well as the networking used to establish secure connections between the database and the application's VPC.
This includes the ability to connect over the cloud provider's private link and private endpoint.

You can choose whether traffic to your Aura instance is allowed to traverse the public internet or if it should use the cloud provider's internal network.
If you enable public traffic, the instance is publicly accessible by anyone with the correct username and password.

For your instance to instead be completely private, use the cloud provider's network and create a private endpoint inside your VPC, which gives you a private connection to Aura.

If you enable *both* private and public traffic, your instance is accessible both privately and publicly.
```

Reply:
Thanks Jessica - I also had a go at re-writing it, wdyt?

```
== Network Access

Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
This includes the ability to connect over the cloud provider's private link and private endpoint.

* If you want your Aura instances to be both private and public, go to the *Aura Console* > *Security* > *Network Access*, then in the network configuration set public and private access.
* If you want your Aura instances to be public, traffic to your Aura instance is allowed to traverse the public internet and the instance is accessible with the username and password. 
Anyone in the world can try to log in if they know the public address.
* If you want your Aura instances to be private, use your cloud provider's internal network. 
To use the cloud provider's network, create a private endpoint inside your VPC. 
This gives you a private connection to Aura. 
If public traffic is turned off then access is completely private and the only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create. 

== Private endpoints

Private endpoints are network interfaces inside your own VPC, which can only be accessed within your private network. 
The cloud provider connects them over their network to Neo4j Aura. 
By design they are not exposed to the public internet, ensuring that critical services are accessible only through private, secure networks.
```

Reply:
On second thoughts, I think a mix of both edits could be nice